assets/src/edit-story/utils/useDoubleClick.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * WordPress dependencies
+ */
+import { useCallback, useEffect, useState } from '@wordpress/element';
+
+// Disable reason: Should not check indentation in code samples inside
+// markdown, but it does.
+
+/* eslint-disable jsdoc/check-indentation */
```

Comment:
Why isn't this working properly? Does WP eslint have weird config for this rule?